From c562c8ce9e9eafdbd6dc37535f85f6e7c86c252b Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Thu, 11 Apr 2019 18:37:03 +0000
Subject: [PATCH] TOOLS-2169 engbld needs to know about 18.4 Reviewed by: Tim
 Foster <tim.foster@joyent.com> Approved by: Tim Foster
 <tim.foster@joyent.com>

---
 tools/validate-buildenv.sh | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/tools/validate-buildenv.sh b/tools/validate-buildenv.sh
index 7aa1f68..ed15251 100755
--- a/tools/validate-buildenv.sh
+++ b/tools/validate-buildenv.sh
@@ -82,6 +82,8 @@ declare -A PKGSRC_MAP=(
     [04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f]=2015Q4
     [1ad363ec-3b83-11e8-8521-2f68a4a34d5d]=2018Q1
     [b6ea7cb4-6b90-48c0-99e7-1d34c2895248]=2018Q1
+    [c2c31b00-1d60-11e9-9a77-ff9f06554b0f]=2018Q4
+    [a9368831-958e-432d-a031-f8ce6768d190]=2018Q4
 )
 
 # Used to provide useful error messages to the user, mapping the
@@ -94,6 +96,8 @@ declare -A SDC_MAP=(
     [04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f]=triton-origin-multiarch-15.4.1@1.0.1
     [1ad363ec-3b83-11e8-8521-2f68a4a34d5d]=minimal-multiarch@18.1.0
     [b6ea7cb4-6b90-48c0-99e7-1d34c2895248]=triton-origin-multiarch-18.1.0@1.0.1
+    [c2c31b00-1d60-11e9-9a77-ff9f06554b0f]=minimal-64-lts@18.4.0
+    [a9368831-958e-432d-a031-f8ce6768d190]=triton-origin-x86_64-18.4.0@master-20190410T193647Z-g982b0ce
 )
 
 # Used to provide useful error messages to the user, mapping the NODE_PREBUILT
@@ -107,6 +111,8 @@ declare -A JENKINS_AGENT_MAP=(
     [04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f]=1356e735-456e-4886-aebd-d6677921694c
     [1ad363ec-3b83-11e8-8521-2f68a4a34d5d]=8b297456-1619-4583-8a5a-727082323f77
     [b6ea7cb4-6b90-48c0-99e7-1d34c2895248]=8b297456-1619-4583-8a5a-727082323f77
+    [c2c31b00-1d60-11e9-9a77-ff9f06554b0f]=29b70133-1e97-47d9-a4c1-e4b2ee1a1451
+    [a9368831-958e-432d-a031-f8ce6768d190]=29b70133-1e97-47d9-a4c1-e4b2ee1a1451
 )
 
 # For each pkgsrc version, set a list of packages that must be present
@@ -172,6 +178,18 @@ PKGSRC_PKGS_2018Q1="
     pcre-8.42
     pigz"
 
+PKGSRC_PKGS_2018Q4="
+    grep
+    build-essential
+    python27
+    py27-expat
+    coreutils
+    gsed
+    gsharutils
+    flex
+    pcre-8.42
+    pigz"
+
 UPDATES_URL="https://updates.joyent.com?channel=experimental"
 UPDATES_IMG_URL="https://updates.joyent.com/images/"
 
-- 
2.21.0

